src/gnome_abrt/views.py

```diff
@@ -989,93 +940,93 @@ def on_gac_delete_activate(self, action):
                 self._remove_problem_from_storage(ex.problem_id)
 
     @handle_problem_and_source_errors
-    def on_gac_detail_activate(self, action):
+    def on_gac_detail_activate(self, parameter, user_data=None):
         selected = self._get_selected(self.lss_problems)
         if selected:
             wrappers.show_problem_details_for_dir(
                     selected[0].problem_id, self)
 
     @handle_problem_and_source_errors
-    def on_gac_report_activate(self, action):
+    def on_gac_report_activate(self, parameter, user_data=None):
         selected = self._get_selected(self.lss_problems)
         if selected and not selected[0]['not-reportable']:
             self._controller.report(selected[0])
 
+    @Gtk.Template.Callback()
     @handle_problem_and_source_errors
     def on_se_problems_search_changed(self, entry):
         self._filter.set_pattern(entry.get_text())
 
     def _on_key_press_event(self, sender, event):
-        return self._builder.search_bar.handle_event(event)
+        return self.search_bar.handle_event(event)
 
     def _hide_problem_filter(self):
-        self._builder.se_problems.set_text("")
-        self._builder.search_bar.set_search_mode(False)
+        self.se_problems.set_text("")
+        self.search_bar.set_search_mode(False)
 
     def _show_problem_filter(self):
-        self._builder.search_bar.set_search_mode(True)
-        self._builder.se_problems.grab_focus()
+        self.search_bar.set_search_mode(True)
+        self.se_problems.grab_focus()
 
+    @Gtk.Template.Callback()
     def on_se_problems_key_press_event(self, sender, data):
         if data.keyval == Gdk.KEY_Escape:
             self._hide_problem_filter()
 
         return False
 
-    def on_gac_search_activate(self, action):
-        if self._builder.search_bar.get_search_mode():
+    def on_gac_search_activate(self, parameter, user_data=None):
+        if self.search_bar.get_search_mode():
             self._hide_problem_filter()
         else:
             self._show_problem_filter()
 
-    def on_gac_opt_all_problems_activate(self, action):
-        conf = config.get_configuration()
-        conf['all_problems'] = self._builder.chb_all_problems.get_active()
-
     def on_gac_control_preferences_activate(self, action):
         wrappers.show_events_list_dialog(self)
 
-    def on_gac_open_directory_activate(self, action):
+    def on_gac_open_directory_activate(self, parameter, user_data=None):
```

Comment:
"parameter"? Even if ignored I guess it's better to give the name in the documentation or something that's more understandable.

Reply:
oh... I see. Also, is user_data really passed? I though it was ignored in python, I might be wrong.